docs/ref/forms/fields.txt

```diff
@@ -299,6 +299,18 @@ as the rendered output.
 See the :ref:`format localization <format-localization>` documentation for
 more information.
 
+``disabled``
+~~~~~~~~~~~~~~~~
+
+.. attribute:: Field.disabled
+
+.. versionadded:: 1.9
+
+The ``disabled`` boolean argument, when set to ``True``, allows displaying a
+form field in a disabled way, that is its widget will have the ``disabled``
+HTML attribute, so it won't be editable by users. Even if some user is tampering
+with the widget value client-side, the form will not consider the value coming
```

Comment:
in this sentence, `form` has 2 meanings: the Django `Form` instance, and the HTML/HTTP form data.
Maybe change `the form will not consider` to something clearer like `the Django Form instance`? Makes it sound a little stilted, though.